koku/api/query_handler.py

```diff
@@ -326,8 +326,8 @@ def filter_to_order_by(self, parameters):  # noqa: C901
             parameters (QueryParameters): The parameters object
 
         """
-        for group_by_key in parameters.parameters['group_by']:
-            for group_by_value in parameters.parameters['group_by'][group_by_key]:
+        for group_by_key in parameters.parameters.get('group_by', {}):
+            for group_by_value in parameters.parameters.get('group_by', {}).get(group_by_key, {}):
```

Comment:
I don't think the default here on line 330 would be a dictionary for the second get. Plus you don't need the protection on the second line as the first loop on 329 checks that `'group_by'` exists and will give you a valid `group_by_value` if you get to 330.  I tried to show this update with my earlier branch before the break: https://github.com/project-koku/koku/commit/e2421db930e70b72a1c5b88f2d8498d800d36d16#diff-2fb28354acc7323fa4d295ec67e80c5aR330

Reply:
Removed the protection from the second line